### DISCUSSION TITLE
网站的当前状态是乱码

### DISCUSSION THREAD

**Discussion by kongku** [1Panel-dev/1Panel#5004] [Q&A]
### 1Panel 版本

社区版:v1.10.7-lts

### 问题描述

升级到v1.10.7-lts后，查看网站的当前状态是https标签乱码，从接口返回的数据也是这样。
![image](https://github.com/1Panel-dev/1Panel/assets/29189288/9f129c3a-1555-4941-a4d0-4ddaed983321)

### 重现步骤

在面板中打开网站->设置->当前状态

**Comment by zhengkunwang223** [comment]
是不是添加了 127.0.0.1 的网站？

**Comment by kongku** [comment]
> 是不是添加了 127.0.0.1 的网站？

把waf关闭后就可以查看了，是什么原因吗？

**Comment by kongku** [comment]
> 是不是添加了 127.0.0.1 的网站？

找到原因了，是因为waf全局设置中添加了Go-http-client的user-agent过滤！

**Comment by kongku** [comment]
> 是不是添加了 127.0.0.1 的网站？

但是，为什么在waf的全局ip白名单中添加了127.0.0.1，然后全局user-agent过滤添加Go-http-client，这个ip白名单好像不起作用，依然会乱码？

**Comment by zhengkunwang223** [comment]
> > 是不是添加了 127.0.0.1 的网站？
> 
> 但是，为什么在waf的全局ip白名单中添加了127.0.0.1，然后全局user-agent过滤添加Go-http-client，这个ip白名单好像不起作用，依然会乱码？

这个状态页面 需要用到 127.0.0.1 这个 server_name 如果你自己也增加了一个 127.0.0.1 的 server_name 就会冲突

**Comment by Seele0oO** [comment]
```
root@nginx:/usr/local/openresty/nginx/conf/conf.d# grep -rl "127.0.0.1" /usr/local/openresty/nginx/conf/conf.d/
/usr/local/openresty/nginx/conf/conf.d/default.conf
```

无论我是否允许这个conf加载，都不能正常显示状态。

**Reply by zhengkunwang223** [reply]
你的异常页面发一下

**Reply by Seele0oO** [reply]
![图片](https://github.com/user-attachments/assets/3014112c-a317-44eb-ab06-ced032982949)
